### DISCUSSION TITLE
Azure ARM template validation output mismatch

### DISCUSSION THREAD

**Discussion by Susanta2000** [aquasecurity/trivy#5056] [Q&A]
### Question

I am validating a azure ARM template file using trivy. When I was validating, trivy gave me 1 issue but when I am validating the same template file using arm-ttk it gave me 2 issues. And issues given by trivy and arm-ttk are different.
**Issue given when validating using arm-ttk:**
1. Location Should Not Be Hardcoded (27 ms) 
2. Resources Should Have Location (7 ms)
**Issue given when validating using trivy:**
1. CRITICAL: Storage account uses an insecure TLS version.
Can anyone explain this?

### Target

None

### Scanner

None

### Output Format

JSON

### Mode

None

### Operating System

Ubuntu

### Version

```bash
0.43.1
```

**Comment by nikpivkin** [comment]
Hi @Susanta2000 !

The output of the tools may not match, as they use different rules. It would be nice if you provided an example configuration.
